packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -92,25 +92,15 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
     event LiquidityRemoved(address indexed token, uint256 amount, uint256 lpTokensBurnt, address liquidityProvider);
     event DepositRelayed(
         uint256 indexed relayId,
-        uint8 chainId,
-        uint64 depositId,
-        address indexed l2Sender,
+        DepositData depositData,
```

Comment:
I'm surprised you don't need to specify this as a bytes32 type. I guess solidity automatically casts it? It might be better to explicitly use bytes32

Reply:
Hmmm, as far as I'm aware the struct gets encoded into the Abi so clients can decide the full struct transparently. I think this would be the same cost as a bytes array, but clients would just have an easier time deciding the info in web3 or ethers.

Maybe I'm misunderstanding your point tho

Reply:
> Hmmm, as far as I'm aware the struct gets encoded into the Abi so clients can decide the full struct transparently. 

Ah I see what you're saying. So the entire `depositData` struct is emitted (cheaply I'm hoping). Didn't know you could do this haha

Reply:
Yeah, it should be the same cost, since we just removed variables from the event and put them into the struct.